### PR TITLE
plot: update some output strings

### PR DESCRIPTION
**Pull request by jorgeorpinel**
* [x] ❗ I have followed the [Contributing to DVC](https://dvc.org/doc/user-guide/contributing/core) checklist.

* [x] 📖 If this PR requires [documentation](https://dvc.org/doc) updates, I have created a separate PR (or issue, at least) in [dvc.org](https://github.com/iterative/dvc.org) and linked it here. If the CLI API is changed, I have updated [tab completion scripts](https://github.com/iterative/dvc/tree/master/scripts/completion).

* [x] ❌ I will check DeepSource, CodeClimate, and other sanity checks below. (We consider them recommendatory and don't expect everything to be addressed. Please fix things that actually improve code or fix bugs.)

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
Only Py36 on Linux build failed (https://github.com/iterative/dvc/pull/3759/checks) Seems like a glitch.

File: dvc/command/plot.py [hunk 1]
```diff
@@ -148,7 +148,7 @@ def add_parser(subparsers, parent_parser):
         "--stdout",
         action="store_true",
         default=False,
-        help="Print plot specification to stdout.",
+        help="Print plot location to stdout.",
```
Comment (pared) [inline dvc/command/plot.py:151]: So this option lets user print the plot as a text to stdout. So, plot by default logs plot location to stdout. When using `--stdout` option, it is the plot file content that will be displayed on stdout.

Comment (jorgeorpinel) [reply dvc/command/plot.py:151]: Oh I see. Got it wrong then, it seems. Let me FTFY ⏳

**Comment by jorgeorpinel** [comment]
Fixed your comment, @pared. I also made a few more changes to match the now published version in https://dvc.org/doc/command-reference/plot/diff as well as a few more small upcoming changes in iterative/dvc.org/pull/1283. PLAT

**Comment by jorgeorpinel** [comment]
I mean PTAL 😆

**Pull request merged**